build_test.go

```diff
@@ -1,4 +1,4 @@
-package pack_test
+package pack
```

Comment:
Was it intentional to make this "whiteboxed"? It doesn't seem like any methods are not exported.